src/flask_pyoidc/pyoidc_facade.py

```diff
@@ -269,10 +269,19 @@ def client_credentials_grant(self):
                                       access_token_required=True)
             auth.init_app(app)
             auth.clients['default'].client_credentials_grant()
+
+        Optionally, you can also specify scope for the access token.
+
+        ::
+
+            auth.clients['default'].client_credentials_grant(
+                scope=['read', 'write'])
         """
         client_credentials_payload = {
             'grant_type': 'client_credentials'
         }
+        if scope:
+            client_credentials_payload['scope'] = ' '.join(scope)
```

Comment:
A minor change, client credentials flow as per ouath specs can also take scope as an argument which takes string as a parameter delimited by spaces to separate multiple scopes. E.g. `scope='read write'`. Just to make it easier for the user, I'm taking it as a list and convert it into the string for the payload.